src/commands/project/list.js

```diff
@@ -0,0 +1,95 @@
+import {sprintf} from 'sprintf-js'
+
+import getServiceBaseURL, {GAME} from '../../util/getServiceBaseURL'
+import errorReporter from '../../util/errorReporter'
+import graphQLFetcher from '../../util/graphQLFetcher'
+import {userIsPlayer} from '../../util/userValidation'
+
+function invokeProjectListWithReviewsAPI(lgJWT) {
+  const query = {
+    query: `
+query {
+  getProjectsWithReviewResponsesForPlayer {
+    name
+    artifactURL
+    projectReviewResponses {
+      name
+      value
+    }
+  }
+}
+    `,
+  }
+  return graphQLFetcher(lgJWT, getServiceBaseURL(GAME))(query)
+    .then(data => data.getProjectsWithReviewResponsesForPlayer)
+}
+
+function invokeProjectListSummaryAPI(lgJWT) {
+  const query = {
+    query: `
+query {
+  getProjectSummaryForPlayer {
+    numActiveProjectsForCycle
+    numTotalProjectsForPlayer
+  }
+}
+    `,
+  }
+  return graphQLFetcher(lgJWT, getServiceBaseURL(GAME))(query)
+    .then(data => data.getProjectSummaryForPlayer)
+}
+
+function formatProjectList(projects) {
+  const numReviewed = projects.filter(proj => (
+    proj.projectReviewResponses.filter(resp => resp.value).length > 0
+  )).length
+  const preface = `You have reviewed ${numReviewed} / ${projects.length} projects this cycle. Nice work!`
+  const fmt = '%-1s  %-21s  %-3s  %-3s  %s'
+  const header = sprintf(fmt, '', 'Project', 'C', 'Q', 'Artifact')
+  const underlines = sprintf(fmt, '', '---------------------', '---', '---', '----------------------------------------')
```

Comment:
It might be easier to tell if these underlines are the right length by using `'_'.repeat(21)` etc. That would even let us pull the column widths out into constants.